tortoise/backends/base/executor.py

```diff
@@ -344,17 +344,22 @@ def get_update_sql(self, update_fields: Optional[List[str]]) -> str:
     async def _prefetch_direct_relation(
         self, instance_list: "List[Model]", field: str, related_query: "QuerySet"
     ) -> "List[Model]":
-        related_objects_for_fetch = set()
+        related_objects_for_fetch = {}
```

Comment:
In  the pydantic branch I have a few fixes for this, basically to preserve ordering. I'll do a release this weekend with a few fixes here. So be prepared for conflicts here.

Reply:
Okay I see. I will resolve conflict after pydantic branch merged.

Reply:
No, I merged the fixes into master (to keep the already huge pydantic work focused).

Reply:
Ah, I understand. I checked it